packages/content/test/_get-content.test.ts

```diff
@@ -0,0 +1,29 @@
+import { getContent } from "../src";
+import * as portalModule from "@esri/arcgis-rest-portal";
+import * as hubModule from "@esri/hub-common";
+
+describe("getContent", () => {
+  it("get content data from portal", async () => {
+    spyOn(hubModule, "getModel").and.returnValue(
+      Promise.resolve({
+        description: "Content from Portal"
+      })
+    );
+
+    const content = await getContent("some-id", {isPortal: true} as hubModule.IHubRequestOptions);
+
+    expect(content.description).toBe("Content from Portal");
+  });
+
+  it("get content data from Hub", async () => {
+    spyOn(hubModule, "getFromHubAPI").and.returnValue(
```

Comment:
jasmine spies have --> `.and.resolveTo(whatever)`. And usually you want to make assertions on the spy, so 
```
const mySpy = spyOn(....);
...later...
expect(mySpy.calls.argsFor(1)[0]).toBe('foo', 'was called with foo');
```